src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,6 +60,8 @@ public RobotContainer() {
     // Configure the button bindings    
 
     configureButtonBindings();
+
+    OI.Driver.getBalanceButton().toggleOnTrue(assistedBalanceCommand); //C on Keyboard
```

Comment:
Move this to the configureButtonBindings method

Reply:
Done